salt/utils/files.py

```diff
@@ -349,6 +349,10 @@ def fopen(*args, **kwargs):
                 )
         except IndexError:
             pass
+    if six.PY2:
+        # encoding is not available in Py2 and will cause an error later if passed
+        if kwargs.pop('encoding', False):
+            log.debug('Remove kwarg "encoding" on Py2')
```

Comment:
There's really no context to this log message, so I feel like it would clutter up the debug logging. We should either improve it or remove it.

We should also look into using `io.open()` instead of `open()`, because that is the same function on both PY2 and PY3.

Reply:
Removed log message